patternfly/patternfly-elements
# fix: unwanted padding on pfe-accordion

## Unwanted padding on pfe-accordion

### Related issue

- (#883) Pfe-accordion: disclosure includes unwanted padding in the panel


### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Preview link](https://5e6089f7c8e38b0008963801--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/) 


### What has changed and why

- Updating padding to match the design specifications


### Testing instructions

In the demo page, inspect the disclosure element, check that the padding is diffenrent between the left and rigth side. Check that the right padding is greater than twice the left padding value. After the changes, the right and left padding values are the same.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
